app/views/layouts/application.html.erb

```diff
@@ -11,7 +11,10 @@
   </head>
 
   <body>
-    <%= yield %>
-    <%= render 'shared/footer'%>
+    <div class="<%= @home_page ? 'c-home__bg' : 'c-container'%>">
```

Comment:
I'd use a view helper to determine this based on the the request url, rather than using the controller